src/publish/confluence/confluence-helper.ts

```diff
@@ -486,9 +608,65 @@ export const updateImagePaths = (body: ContentBody): ContentBody => {
   return body;
 };
 
-export const findAttachments = (bodyValue: string): string[] => {
+export const findAttachments = (
+  bodyValue: string,
+  publishFiles: string[] = [],
+  filePath: string = ""
+): string[] => {
+  const pathList = filePath.split("/");
```

Comment:
Another file path question - if this were on a windows system would this work? There is a `dirname` which I believe would do this without regard to path separators if that is needed.

Reply:
(If this is a change you think should be made, there are other similar patterns that I haven't highlighted since I don't want to make unnecessary noise if it is ok)

Reply:
It may be wise for us to talk about this offline.  I think I can do a better job with my path work here.

Reply:
https://github.com/rstudio/quarto-confluence-test/issues/36